hal/README.md

```diff
@@ -134,11 +134,41 @@ The object provided to the Mustache template for automatically-generated documen
 }
 ```
 
+## Utilities `@azure-iot/hal/util`
+
+### Server
+Server-specific utility methods can be accessed by calling the utility import as a function on the server object, eg. `util(server).method(...)`.
+
+#### `links(rel, [params])`
```

Comment:
i would call this hrefs for consistency with the api below. I also think that you should make the params optional in both